executor/common_linux.h

```diff
@@ -205,6 +206,36 @@ static int netlink_send(struct nlmsg* nlmsg, int sock)
 	return netlink_send_ext(nlmsg, sock, 0, NULL);
 }
 
+static int netlink_query_family_id(struct nlmsg* nlmsg, int sock, const char* family_name)
```

Comment:
I think syz_genetlink_get_family_id pseudo-syscall also doing the same, so potentially can be refactored to use this function as well.

Reply:
Refactored that function in a separate commit